Diagnostics/HealthChecker/Analyzer/Security/Invoke-AnalyzerSecuritySettings.ps1

```diff
@@ -223,4 +223,19 @@ Function Invoke-AnalyzerSecuritySettings {
     Invoke-AnalyzerSecurityExchangeCertificates -AnalyzeResults $AnalyzeResults -HealthServerObject $HealthServerObject -DisplayGroupingKey $keySecuritySettings
     Invoke-AnalyzerSecurityAMSIConfigState -AnalyzeResults $AnalyzeResults -HealthServerObject $HealthServerObject -DisplayGroupingKey $keySecuritySettings
     Invoke-AnalyzerSecurityMitigationService -AnalyzeResults $AnalyzeResults -HealthServerObject $HealthServerObject -DisplayGroupingKey $keySecuritySettings
+
+    if ($HealthServerObject.ExchangeInformation.BuildInformation.AffectedByFIPFSUpdateIssue) {
+        $AnalyzeResults | Add-AnalyzedResultInformation -Name "FIP-FS Update Issue Detected" -Details $true `
+            -DisplayGroupingKey $keySecuritySettings `
+            -DisplayWriteType "Red"
+
+        $AnalyzeResults | Add-AnalyzedResultInformation -Details "More Information: https://aka.ms/HC-FIPFSUpdateIssue" `
+            -DisplayGroupingKey $keySecuritySettings `
+            -DisplayWriteType "Red" `
+            -DisplayCustomTabNumber 2
+    } elseif ($null -eq $HealthServerObject.ExchangeInformation.BuildInformation.AffectedByFIPFSUpdateIssue) {
```

Comment:
This check isn't going to work because you provided the class type to be a `bool` which can only be `true` or `false`

![image](https://user-images.githubusercontent.com/22776718/147955431-247ae5b6-7209-45c3-82ca-c0ce81a86828.png)

Reply:
Fixed with the upcoming commit.